internal/gitsync/github/release_syncer_test.go

```diff
@@ -0,0 +1,71 @@
+package github
```

Comment:
these tests are questionable as well, maybe a bit better than the handler tests.

Ideally the test would be show that after calling `UpdateOrInstallFromChart`, we can observe that the release "exists" regardless of whether it was installed from nothing, or upgraded from an existing release. That'd be a much better behavioural test that what i'm doing here, checking that the right helm client methods are called.

Using a mock `helmClient` to test that the release exists seems pointless, I'm basically testing the mock at that point. So I'm not sure what a better way to write these tests would be.